thinc/tests/layers/test_mappers.py

```diff
@@ -0,0 +1,59 @@
+from thinc.layers import premap_ids, remap_ids, remap_ids_v2
+import numpy as np
+
+
+def get_keys():
+    return np.array([4, 2, 6, 1, 8, 7, 9, 3, 30])
+
+
+def get_mapper():
+    keys = get_keys()
+    return {int(k): int(v) for v, k in enumerate(keys)}
+
+
+def test_premap():
+    mapper = get_mapper()
+    keys = get_keys()
+    premap = premap_ids(mapper, default=99)
+    values, _ = premap(keys, False)
+    assert all(values.squeeze() == np.asarray(range(len(keys))))
```

Comment:
Rather than `all(True)` (with a potential for implicit conversions and python comparisons to mask bugs), for these kinds of comparisons in tests I'd prefer to use `numpy.testing.assert_equal` to directly compare the numpy arrays.